tools/osbuilder/node-builder/azure-linux/uvm_install.sh

```diff
@@ -29,13 +29,48 @@ mkdir -p "${UVM_PATH}"
 
 echo "Installing UVM files to target directory"
 if [ "${CONF_PODS}" == "yes" ]; then
-	cp -a --backup=numbered "${IGVM_FILE_NAME}" "${UVM_PATH}"
-	cp -a --backup=numbered "${IGVM_DBG_FILE_NAME}" "${UVM_PATH}"
-	cp -a --backup=numbered "${UVM_MEASUREMENT_FILE_NAME}" "${UVM_PATH}"
-	cp -a --backup=numbered "${UVM_DBG_MEASUREMENT_FILE_NAME}" "${UVM_PATH}"
+	CONF_PODS_INSTALL_SUCCESS=false
+	if [ -f "${IGVM_FILE_NAME}" ]; then
```

Comment:
New: allow script to continue if only one of the release/debug configurations is built. Fail if neither is present.